Overview.src.html

```diff
@@ -202,6 +202,34 @@ <h2 class="short">Conformance</h2>
 <span>process request body</span> for <var>request</var>.
 <!-- XXX xref "read", "payload body" -->
 
+<h3>Token Binding</h3>
+
+<p>The technique of Token Binding allows user agents to work with servers to protect credentials
```

Comment:
I'd suggest rewording this section a bit. Perhaps something like:

> ```
> In order to protect security tokens like <a>credentials</a> and OAuth tokens, user agents
> and servers can use <dfn>token binding</dfn> to cryptographically associate a given
> security token with a particular TLS connection, thereby making it more difficult for
> attackers to abuse the capabilities the security token grants. This technique is described in
> detail in [[TOKBIND-NEGOTIATION]], [[TOKBIND-PROTOCOL]], and [[TOKBIND-HTTPS]].
>
> A <dfn>token binding ID</dfn> is the representation of a token binding, as described in
> <a href="https://tools.ietf.org/html/draft-ietf-tokbind-protocol#section-3.2">section 3.2</a>
> of [[TOKBIND-PROTOCOL]].
> ```

I don't think you need to define "token binding message", as you only use it when constructing the `Sec-Token-Binding` header that we'll talk about in a minute.